spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -107,10 +107,10 @@ def process(options)
       context "when process is interupted" do
         before do
           stub_const("Interupter", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              Process.kill("INT", 0)
-            end
-          })
+                                   def process(options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.